### ISSUE TITLE
Ekomind MU Bug

### ISSUE THREAD

**Issue by konals** [mtgred/netrunner#2829]
I've been playing a couple games with Ekomind and I've noticed that I seem to have more MU than I should.

I just finished a game where I had Ekomind installed with 15 cards in hand, and two LLDS Memory Diamonds.

This should have given me 17 MU. I also had 14 MU of programs installed, which should have brought me to a total of 3 MU, instead I was at 15 MU free.

I'm not sure exactly what causes the issue but I think Ekomind is assigning extra memory for some reason.

I did drag a card back to my hand that I didn't intend to discard at one point.

**Comment by JoelCFC25** [comment]
Yeah, this is actually a longstanding problem--a very little used card which is probably why no one's bothered to say anything. Ekomind's watch state is actually adjusting what your _maximum_ MU is, but is not accounting in any way for what MU is being consumed by installed programs.

**Comment by NoahTheDuke** [comment]
@Saintis, does your MU refactor handle this?

**Comment by Saintis** [comment]
Yeah, this should fix the Ekomind issue
